server/loopbackconnectorsqlite3.js

```diff
@@ -0,0 +1,43 @@
+var sqlite3 = require('loopback-connector-sqlite3').verbose();
+
+
+function dosqlite3(result, callback) {
+  var db = new sqlite3.Database(':memory:');
```

Comment:
@cgole @kraman Am I missing something? This appears to not use the loopback-connector API, and instead use the underlying sqlite3 API directly, so it won't show up in DOA. Shouldn't the connector exerciser use the connector API? Its not documented, but looking at the tests, the API looks more like this: https://github.com/strongloop/loopback-connector-sqlite3/blob/master/test/sqlite.test.js#L32